storage/store.go

```diff
@@ -933,10 +933,36 @@ func (s *Store) BootstrapRange() error {
 	value := proto.Value{}
 	value.SetInteger(int64(sql.MaxReservedDescID + 1))
 	value.InitChecksum(key)
-	if err := engine.MVCCPut(batch, nil, key, now, value, nil); err != nil {
+	if err := engine.MVCCPut(batch, ms, key, now, value, nil); err != nil {
 		return err
 	}
 
+	// Setup the system database, the namespace table and the descriptor table
+	// entries.
+	systemData := []struct {
+		parentID sql.ID
+		name     string
+		id       sql.ID
+		desc     gogoproto.Message
```

Comment:
Yeah, it is slightly icky that this is only a `gogoproto.Message`. Better than export `sql.descriptorProto` would be to find a way to move this setup logic into the `sql` package. The challenge there is that we don't want the `sql` package to depend on `storage/engine`. I'll add a TODO to figure something out.